src/appkit.cs

```diff
@@ -10234,7 +10450,7 @@ partial interface NSLayoutManager : NSCoding {
 		[Export ("intAttribute:forGlyphAtIndex:")]
 		nint IntAttributeforGlyphAtIndex (nint attributeTag, nint glyphIndex);
 
-		// TODO: bind this with a safe version
+		[Deprecated (PlatformName.MacOSX, 10, 13)]
```

Comment:
it's `[Internal]`, has the caller(s) been decorated too ?
if there's none then it can simply be removed